streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -44,6 +44,40 @@
 
 /**
  * Factory for creating state stores in Kafka Streams.
+ * <p>
+ * For high-level DSL, i.e., {@link org.apache.kafka.streams.StreamsBuilder StreamsBuilder}, users create
+ * {@link StoreSupplier}s that can be further customized via
+ * {@link org.apache.kafka.streams.kstream.Materialized Materialized}.
+ * For example, a topic read as {@link org.apache.kafka.streams.kstream.KTable KTable} can be materialized into an
+ * in-memory store with custom key/value serdes and caching disabled:
+ * <pre>{@code
+ * StreamsBuilder builder = new StreamsBuilder();
+ * KeyValueBytesStoreSupplier storeSupplier = Stores.inMemoryKeyValueStore("queryable-store-name");
```

Comment:
Why "queryable-store-name"?  IIRC we don't say "queryable store" anywhere else in the docs -- we use the term "interactive queries", if anything.

Reply:
We use parameter name `KStreamBuilder#table(String topic, String queryableStoreName)` in old API: https://github.com/apache/kafka/blob/trunk/streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java#L400